diff --git a/etc/inittab b/etc/inittab
--- a/etc/inittab
+++ b/etc/inittab
@@ -53,11 +53,11 @@
 # Format:
 #  <id>:<runlevels>:<action>:<process>
 1:234:respawn:/sbin/mingetty --noclear tty1
-2:2345:respawn:/sbin/mingetty tty2
-3:2345:respawn:/sbin/mingetty tty3
-4:2345:respawn:/sbin/mingetty tty4
-5:2345:respawn:/sbin/mingetty tty5
-6:2345:respawn:/sbin/mingetty tty6
+2:2345:respawn:/usr/bin/slogin tty2
+3:2345:respawn:/usr/bin/slogin tty3
+4:2345:respawn:/usr/bin/slogin tty4
+5:2345:respawn:/usr/bin/slogin tty5
+6:2345:respawn:/usr/bin/slogin tty6
 
 # Example how to put a getty on a serial line (for a terminal).
 #
