diff --git a/etc/openssh/sshd_config b/etc/openssh/sshd_config
--- a/etc/openssh/sshd_config
+++ b/etc/openssh/sshd_config
@@ -14,6 +14,7 @@
 #AddressFamily any
 #ListenAddress 0.0.0.0
 #ListenAddress ::
+ListenAddress 127.0.0.1
 
 #HostKey /etc/openssh/ssh_host_rsa_key
 #HostKey /etc/openssh/ssh_host_ecdsa_key
@@ -53,7 +54,7 @@
 #IgnoreRhosts yes
 
 # To disable tunneled clear text passwords, change to no here!
-#PasswordAuthentication yes
+PasswordAuthentication no
 #PermitEmptyPasswords no
 
 # Change to yes to enable s/key passwords
